src/Interfaces.ts

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+//
+// MIT License:
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+import * as Msal from 'msal';
+
+type UserInfoCallback = (token: IUserInfo) => void;
+
+interface IUserInfo {
+  jwtAccessToken: string,
+  jwtIdToken: string,
+  user: Msal.User,
```

Comment:
Given that we're creating an underlying abstraction, does it make sense to have a user interface as well? If not in this PR, perhaps future work? Otherwise looks great!

Reply:
Yes, I think so! I am not sure if that is a requirement for getting tests up and running yet, and I was trying to get the smallest change in since refactors are hard to review but ideally all these Msal constructs are abstracted out

Reply:
awesome, sounds good and yes, I suspect testing would surface this :slightly_smiling_face: 

My guess you'll end up crossing this bridge, otherwise maybe open another issue afterwards?

Reply:
I'll go ahead and make the issue now and we can won't fix it later if necessary

Reply:
#46